src/__tests__/02b-onchain-receive.ts

```diff
@@ -43,7 +45,8 @@ beforeEach(async () => {
   initBlockCount = await bitcoindDefaultClient.getBlockCount()
   initialBalanceUser0 = (await walletUser0.getBalances()).BTC
 
-  amount_BTC = +(1 + Math.random()).toPrecision(9)
+  // amount_BTC = +(1 + Math.random()).toPrecision(9)
```

Comment:
Why are we switching to constant amount_BTC instead of random?

Reply:
had set a fix amount for debugging stuff. eventually we should set the random seed to always return the same result.